adafruit_dotstar.py

```diff
@@ -204,17 +205,17 @@ def show(self):
         # Create a second output buffer if we need to compute brightness
         buf = self.buf
         if self.brightness < 0.99:
```

Comment:
Why not `self.brightness < 1.0`? The user may want to use tiny changes in brightness.

Reply:
Done. I was worried about rounding issues with 1.0 but I don't think it'll be a problem.

Reply:
1.0 is represented exactly, so it's OK.